polonius-engine/src/facts.rs

```diff
@@ -64,29 +64,6 @@ pub struct AllFacts<R: Atom, L: Atom, P: Atom, V: Atom, M: Atom> {
     pub path_accessed_at: Vec<(M, P)>,
 }
 
-impl<R: Atom, L: Atom, P: Atom, V: Atom, M: Atom> Default for AllFacts<R, L, P, V, M> {
```

Comment:
We did not do this because `derive(Default)` adds the requirement that (e.g.) `R: Atom + Default`, which we don't want.